docs/data/material/getting-started/example-projects/example-projects.md

```diff
@@ -8,17 +8,9 @@ You can find some example projects in the [GitHub repository](https://github.com
 
 <!-- #default-branch-switch -->
 
-- [Next.js](https://github.com/mui/material-ui/tree/master/examples/material-next) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-next-ts))
-- [Create React App](https://github.com/mui/material-ui/tree/master/examples/material-cra) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-cra-ts))
-- [Remix](https://github.com/mui/material-ui/tree/master/examples/material-remix-ts)
-- [Gatsby](https://github.com/mui/material-ui/tree/master/examples/material-gatsby)
-- [Preact](https://github.com/mui/material-ui/tree/master/examples/material-preact)
-- [CDN](https://github.com/mui/material-ui/tree/master/examples/material-via-cdn)
-- [Express.js (server-rendered)](https://github.com/mui/material-ui/tree/master/examples/material-express-ssr)
-- [Tailwind CSS](https://github.com/mui/material-ui/tree/master/examples/material-cra-tailwind-ts)
-- [Vite.js](https://github.com/mui/material-ui/tree/master/examples/material-vite) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-vite-ts))
-- [Use styled-components as style engine](https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components-ts))
-- [Next.js + @mui/styles (v4 migration helper)](https://github.com/mui/material-ui/tree/master/examples/material-next-ts-v4-v5-migration)
+{{"component": "docs/src/modules/components/ExampleCollection.js"}}
```

Comment:
Should we update the rest of the docs to use the full import path?

<img width="331" alt="Screenshot 2023-05-25 at 12 03 33" src="https://github.com/mui/material-ui/assets/3165635/457499a3-9a49-47d3-9ca9-e98d921cc645">

Reply:
I'm coming from the change in https://github.com/mui/material-ui/pull/37007/files#diff-8896d42292e63fa8530b9808a333b4770fee0e9fbe33d57e0b4ab5b3b723ed3dR223 which seems unnecessary

Reply:
Oh actually, the location of the demo. This is now with the rest of the global component. If this component in only used on this page, it shouldn't be in `docs/src/modules/components/`. This folder is meant for global components. Here, we can host it in the same folder as the docs, so that if we ever refactor it, we would remove this dead demo at the same time.

So we move this file inside `docs/data/material/getting-started/example-projects/`?

Reply:
I'll leave it up to @siriwatknp to decide on this one, then, as he pushed that commit on!